packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -215,6 +217,90 @@ export type {
   User,
 } from "@auth/core/types"
 
+type SignInParams = Parameters<App.Locals["signIn"]>
+async function signIn(
```

Comment:
Nit: Should we move `signIn` and `signOut` to a different file (maybe actions.ts)?